### DISCUSSION TITLE
Kafka connect - getting Connect Timed Out for Google sink connectors

### DISCUSSION THREAD

**Discussion by alexanghel23** [strimzi/strimzi-kafka-operator#6207] [Q&A]
Hello,

Please help me sort out an error i keep getting when trying to connect a topic to Google Bigquery or Google Cloud Storage.
I am running the Strimzi operator in a Kubernetes cluster behind a proxy. I have followed [this](https://strimzi.io/docs/operators/latest/using.html#proc-configuring-proxy-config-cluster-operator-str) guide and set up the https_proxy and http_proxy env variables in the strimzi operator deployment yaml.

However, when launching the connectors (io.aiven.kafka.connect.gcs.GcsSinkConnector, com.wepay.kafka.connect.bigquery.BigQuerySinkConnector) i keep getting this error:
[error.txt](https://github.com/strimzi/strimzi-kafka-operator/files/7881417/error.txt)

I have also set up the proxy env variables in the Kafka connect cluster. Any ideas for a fix?

Thank you!

**Comment by scholzj** [comment]
Strimzi will pass the proxy envirnment variables to the operands. But it cannot guarantee that the connectors will use them. So I guess that could be something to verify with the authors of the connector.
